### PR TITLE
release new rmq

### PR DESCRIPTION
**Pull request by michaelwang13**
Releasing the latest version of RabbitMQ 3.10 3.11 and 3.12.

Updating Alpine versions as well, to use the correct Erlang version (>25 for 3.11, and >26 for 3.12).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +37,9 @@ startup time.
 
 ## Available Versions (Tags)
 
-* `latest`: Currently RabbitMQ 3.9
+* `latest`: Currently RabbitMQ 3.12
+* `3.12`: RabbitMQ 3.12.12
+* `3.11`: RabbitMQ 3.11.28
```
Comment (UserNotFound) [inline README.md:42]: Do we not intend to list 3.10?
Also, even 3.11 is EOL, so i they're listed here we should note as much.
Maybe even link to https://endoflife.date/rabbitmq

File: bin/run-database.sh [hunk 2]
```diff
@@ -124,14 +124,17 @@ if [[ "$1" == "--initialize" ]]; then
 
     rabbitmq-server &
     rmq_pid="$!"
-
+    # potentially need to wait for the server to start running rabbit
+    sleep 10
```
Comment (UserNotFound) [inline bin/run-database.sh:128]: I'm unclear on what the purpose of a sleep here is? All the commands here use `with_retry`, which already includes sleeping and retrying if they fail, so it should already be able to handle the server being unavailable for 2*30 seconds.

**Review by UserNotFound — APPROVED** [review]
Just a couple of comments, but nothing blocking. Since there are TLS version support changes, we should make sure those are reflected in our documentation.

File: bin/run-database.sh (hunk 2 above)
Comment (michaelwang13) [reply bin/run-database.sh:128]: whoops - added that in my struggle debug of TLS1.3 - removing.

File: README.md (hunk 1 above)
Comment (michaelwang13) [reply README.md:42]: done - Ben and I think we shouldn't even make 3.10 and 3.11 available, since they are EOL already.

**Review by UserNotFound — APPROVED** [review]
LGTM!

**Pull request merged**